README.md

```diff
@@ -20,15 +20,15 @@ Once you've run the above commands the pre-commit framework will be installed wi
 
 ## Usage
 
-The first time you run `git commit` **in a repository** will throw a warning to tell you that you need to create a secrets baseline, as shown below:
+The first time you run `git commit` in a repository will throw a warning to tell you that you need to create a secrets baseline, as shown below:
```

Comment:
I feel this reads better as 'the first time you run git commit in a repository **it** will throw.....

Reply:
Also, I know you didn't change this bit but can you also change the sentence above as a drive by fix?
"Once you've run the above commands the pre-commit framework will be installed with the detect-secrets plugin added to**its** config globally for git."
(ie change it's -> its)